lib/seattleflu/api/datastore.py

```diff
@@ -156,6 +156,42 @@ def store_redcap_det(session: DatabaseSession, document: str) -> None:
             raise BadRequestDatabaseError(error) from None
 
 
+@export
+@catch_permission_denied
+def export_metadata_for_augur_build(session: DatabaseSession) -> dict:
+    """
+    Export metadata for augur build from shipping view
+    """
+    with session, session.cursor() as cursor:
+        cursor.execute("select * from shipping.metadata_for_augur_build_v1")
+        return list(cursor)
+
+
+@export
+@catch_permission_denied
+def export_genomic_data(session: DatabaseSession,
```

Comment:
I think it's nice to avoid the very generic "data" and use a more precise noun when possible. Something like `fetch_genomic_sequences` maybe?